mitodl/micromasters
# Add ARIA labels to sharing buttons

#### What are the relevant tickets?
Fixes #1687.

#### What's this PR do?
Adds [ARIA labels](http://rawgit.com/w3c/aria/master/aria/aria.html#aria-label) to the share buttons.

#### How should this be manually tested?
Enable VoiceOver or ChromeVox, and tab over to the share buttons. Make sure that the screen reader reads out an understandable description of each link.
